neurons/validator.py

```diff
@@ -956,6 +977,10 @@ def log_step(
             "uids": uids,
             "uid_data": {},
         }
+
+        # The sub-competition weights
+        sub_competition_weights = torch.softmax(model_weights / constants.temperature, dim=0)
+
         for uid in uids:
             step_log["uid_data"][str(uid)] = {
                 "uid": uid,
```

Comment:
Did you need to add the competition_weight to the step log as well for the leaderboards?

Reply:
I think so, it should help filtering the back end data.